<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>800</width>
     <height>400</height>
    </rect>
   </property>
   <property name="title">
    <string>필요물품</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QScrollArea" name="scrollArea_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>760</width>
      <height>240</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>758</width>
       <height>238</height>
      </rect>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>필요물품 목록들</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_17">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>611</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_31">
    <item>
     <widget class="QPushButton" name="pushButton_27">
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_28">
      <property name="text">
       <string>삭제</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string>수령확인</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_29">
      <property name="text">
       <string>완료</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
